.github/workflows/deploy.yml

```diff
@@ -23,30 +23,13 @@ jobs:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
 
-      - name: Clone apisix's docs from repo & update image's link
```

Comment:
@juzhiyuan may I ask the reason to remove APISIX docs? They are useless, or have other alternatives?

Reply:
because currently, docs from `apisix/doc` need to be adjusted in many places(like images position) to have a better view (not good though), will add them back.

https://github.com/apache/apisix-website/pull/162/files#diff-28802fbf11c83a2eee09623fb192785e7ca92a3f40602a517c011b947a1822d3L35-L45

Reply:
ok. Thanks for the tip.